src/http.cc

```diff
@@ -1933,6 +2057,39 @@ HttpStateData::httpBuildRequestHeader(HttpRequest * request,
     strConnection.clean();
 }
 
+static const char *
+mkUpgradeHeader(HttpStateData *state, HttpRequest *req, const String &value)
```

Comment:
It can, however I prefer to still use String, just because String uses the HttpMsg object.  To avoid conversions. Also it is a Static code inside http.cc

Again if you believe that it is important I can change it to SBuf.